### ISSUE TITLE
Include tests in sdist

### ISSUE THREAD

**Issue by jayvdb** [ulope/pytest-select#2]
It would be helpful if the sdist included the tests, so that they can be run when building an RPM, like I am doing at https://build.opensuse.org/package/show/home:jayvdb:coala:test-rig/python-pytest-select

The same problem exists in poetry itself.
c.f. https://github.com/sdispater/poetry/issues/1109

**Comment by ulope** [comment]
The problem with that is that the tests have additional dependencies.

Can't you use the tag archive from the GitHub releases to base your package on?

**Comment by jayvdb** [comment]
I dont mind if I need to manage the extra dependencies myself.  The important bit is getting a versioned tarball with a `setup.py` and the tests together.

**Comment by ulope** [comment]
> The important bit is getting a versioned tarball with a setup.py and the tests together.

Yeah ok, so as I asked above, why not use the tag archives from https://github.com/ulope/pytest-select/releases?
